### ISSUE TITLE
Cannot cast java.lang.String to org.knowm.xchart.style.Theme trying to create a chart

### ISSUE THREAD

**Issue by skovsboll** [krukow/vsts-flow-metrics#9]
I can get the raw data out:

```
$ ./flow-metrics cycle-time cache/2018-04-11T08:10-closed-stories-15d.wiql.json
{
  "35696" : 7.125,
  "36686" : null,
  "36493" : 0.875,
  "36306" : 1.0416666666666667,

  ...

  "36857" : 0.0,
  "34908" : null,
  "36425" : 0.0,
  "37027" : 0.125,
  "35673" : 11.166666666666666,
  "37022" : 0.8333333333333334,
  "36856" : 0.0,
  "32935" : 6.625,
  "36497" : 2.2916666666666665
}
```

but the chart generation fails:

```
$ ./flow-metrics cycle-time cache/2018-04-11T08:10-closed-stories-15d.wiql.json --chart test.svg
#error {
 :cause Cannot cast java.lang.String to org.knowm.xchart.style.Theme
 :via
 [{:type java.lang.ClassCastException
   :message Cannot cast java.lang.String to org.knowm.xchart.style.Theme
   :at [java.lang.Class cast Class.java 3369]}]
 :trace
 [[java.lang.Class cast Class.java 3369]
  [clojure.lang.Reflector boxArg Reflector.java 427]
  [clojure.lang.Reflector boxArgs Reflector.java 460]
  [clojure.lang.Reflector invokeMatchingMethod Reflector.java 58]
  [clojure.lang.Reflector invokeInstanceMethod Reflector.java 28]
  [com.hypirion.clj_xchart$category_chart_STAR_ invokeStatic clj_xchart.clj 424]
  [com.hypirion.clj_xchart$category_chart_STAR_ invoke clj_xchart.clj 406]
  [com.hypirion.clj_xchart$category_chart invokeStatic clj_xchart.clj 510]
  [com.hypirion.clj_xchart$category_chart invoke clj_xchart.clj 497]
  [vsts_flow_metrics.charts$view_cycle_time invokeStatic charts.clj 40]
  [vsts_flow_metrics.charts$view_cycle_time invoke charts.clj 26]
  [vsts_flow_metrics.cli$cycle_time invokeStatic cli.clj 86]
  [vsts_flow_metrics.cli$cycle_time invoke cli.clj 72]
  [vsts_flow_metrics.cli$_main$fn__17046 invoke cli.clj 221]
  [vsts_flow_metrics.cli$_main invokeStatic cli.clj 217]
  [vsts_flow_metrics.cli$_main doInvoke cli.clj 192]
  [clojure.lang.RestFn applyTo RestFn.java 137]
  [vsts_flow_metrics.cli main nil -1]]} Uncaught throwable
```

I build from code using java 1.8.0_144 and Leiningen 2.8.1.

**Comment by krukow** [comment]
I can't repro :( 

@skovsboll can you show me the config (show-config) and the WIQL query used for cache.

**Comment by skovsboll** [comment]
Here's the...

[wiql](https://www.dropbox.com/s/q4h9k5xtf711kup/closed-stories-15d.wiql?dl=1)
[config](https://www.dropbox.com/s/9v3f6q32nenuhtr/soren-flow-config.json?dl=1)
[cache](https://www.dropbox.com/s/wo493p44ffrhbbk/2018-04-11T10%3A45-closed-stories-15d.wiql.json?dl=1)
[raw cycle-time output json](https://www.dropbox.com/s/djo6xex1iqh1gpo/raw-cycle-time-output.json?dl=1)

Notice the many nulls in the latter file. Could it be causing problems with the chart component?

**Comment by krukow** [comment]
Fixed with #12 
@skovsboll Note that project is just the first segment of the area path for a team, e.g., just "Mobile-Center" not "Mobile-Center\Team-A".  In your case you don't need to set the project in the config since you're also on Mobile-Center. 

You can control which work items you're targeting with the WIQL file, e.g.,

```
WHERE
        [System.TeamProject] = @project
        AND [System.AreaPath] UNDER "Mobile-Center\Soren-Team"
        AND [System.WorkItemType] IN ("User Story")
        AND [Microsoft.VSTS.Common.ClosedDate] >= @Today - 30
        AND [System.State] IN ("Closed")
```

**Comment by krukow** [comment]
@skovsboll fixed with v1.0.4 release https://github.com/krukow/vsts-flow-metrics/releases/tag/v1.0.4
